### PR TITLE
Add diagram of the game

### LINKED ISSUE #32: Add diagram of the full system
In `game overview` section, add a diagram of the full system: 
   - two EDs
   - EMS

(Reviewer 3 - Comment 3)

### PR DESCRIPTION
**Pull request by MichalisPanayides**
Address comment 3 by reviewer 3:
- Adding a diagram of the whole game theoretic model (similar to the queueing one)

### CODE REVIEW

**Comment by MichalisPanayides** [comment]
image

File: imgs/game_theoretic_model_diagram/main.tex [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            % Others lines
+            \draw (4.2, \diff + 1.8cm) -- +(-169.5pt,0) node[left]
+            {\( \lambda_1^B \)};
+            \draw (4.2, \diff + 1.8cm) -- (4.2, \diff + 0.525);
+            \draw[->] (4.2, \diff + 0.525cm) -- (5, \diff + 0.525cm);
+        \end{tikzpicture}
+    \end{minipage}
+    \caption{A diagrammatic representation of the game theoretic model.
+    Patients arrive at the EMS at a rate of \( \lambda_2 \) and then a
+    proportion of them is distributed to hospitals A (\(p_A\)) and the
```
Comment (drvinceknight) [inline imgs/game_theoretic_model_diagram/main.tex]:
```suggestion
    proportion of them are distributed to hospitals A (\(p_A\)) and the
```

I think?

File: imgs/game_theoretic_model_diagram/main.tex [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    remaining proportion to hospital B (\(p_B\)) where \(p_A + p_B = 1\).
```
Comment (drvinceknight) [inline imgs/game_theoretic_model_diagram/main.tex]:
```suggestion
    remaining proportion to hospital B (\(p_B\)) so that \(p_A + p_B = 1\).
```

**Comment by drvinceknight** [comment]
I like that diagram very much @11michalis11. :+1:

File: imgs/game_theoretic_model_diagram/main.tex (hunk 1 above)
Comment (MichalisPanayides) [reply imgs/game_theoretic_model_diagram/main.tex]: ¯\\_(ツ)_/¯

No idea. I am accepting the suggested change based on a very quick google search

**Pull request merged**